skpro/datatypes/_check.py

```diff
@@ -104,15 +104,9 @@ def check_is_mtype(
     msg_return_dict: str, one of ``"list"`` or ``"dict"``, optional, default="dict"
         whether returned msg, if returned, is a str, dict or list
 
-        * if ``msg_return_dict="list"``,
-          returned ``msg`` is ``str`` if ``mtype`` is ``str``,
-          returned ``msg`` is ``list`` of ``str`` if ``mtype`` is ``list``
+        *  if "list", msg is str if mtype is str, list of str if mtype is list
```

Comment:
I think the docstring was better in `skpro` - could you instead revert changes and apply these changes to `sktime`?

Reply:
will make a pr in `sktime` to make this change